app/views/layouts/shared/_banner.html.erb

```diff
@@ -1,20 +1,7 @@
-<div class="flash flash-success alert alert-dismissible fade show" role="alert">
-  <span><strong>Yay!</strong> 🎉 you successfully signed in to our service.</span>
-  <a data-dismiss="alert" aria-label="Close">
-    <i class="fas fa-times"></i>
-  </a>
-</div>
-
-<div class="flash flash-warning alert alert-dismissible fade show" role="alert">
-  <span><strong>Mmh</strong> 🤔 seems like you don't have <a href="#">profile picture</a> yet.</span>
-  <a data-dismiss="alert" aria-label="Close">
-    <i class="fas fa-times"></i>
-  </a>
-</div>
```

Comment:
Do we want to delete all of the code for default alerts?

Reply:
We will keep the code commented out in _banner.html.erb